cloudant-client/src/main/java/com/cloudant/client/api/CloudantClient.java

```diff
@@ -323,6 +325,40 @@ public MetaInformation metaInformation() {
         return replicator;
     }
 
+    /**
+     * Lists replication jobs. Includes replications created via /_replicate endpoint as well as
+     * those created from replication documents. Does not include replications which have
+     * completed or have failed to start because replication documents were malformed. Each job
+     * description will include source and target information, replication id, a history of
+     * recent event, and a few other things.
+     *
+     * @return List of replication jobs
+     */
+    public SchedulerJobsResponse schedulerJobs() {
+        return couchDbClient.schedulerJobs();
+    }
+
+    /**
+     * Lists replication document states. Includes information about all the documents, even in
+     * completed and failed states. For each document it returns the document ID, the database,
+     * the replication ID, source and target, and other information.
+     *
+     * @return List of replication document states
+     */
+    public SchedulerDocsResponse schedulerDocs() {
+        return couchDbClient.schedulerDocs();
+    }
+
+    /**
+     * Get replication document state for a given replication document ID.
+     *
+     * @param docId The replication document ID
+     * @return Replication document state
+     */
+    public SchedulerDocsResponse.Doc schedulerDocs(String docId) {
```

Comment:
As per earlier comment, this should probably be the singular `schedulerDoc` or rolled into the other method

Reply:
Done in 3946f8e